.travis.yml

```diff
@@ -20,6 +20,7 @@ before_script:
   - GO111MODULE=off go get golang.org/x/tools/cmd/goimports
   - GO111MODULE=off go get github.com/fzipp/gocyclo
   - GO111MODULE=off go get golang.org/x/lint/golint
+  - GO111MODULE=off go get golang.org/x/image/draw
```

Comment:
This should not be needed - we only need to get tools that will be used in the built process.
Dependencies are handled by `go mod` or the `vendor` folder.